addons/im_livechat/static/src/core/web/livechat_channel_info_list.xml

```diff
@@ -2,6 +2,9 @@
 <templates xml:space="preserve">
     <t t-name="im_livechat.channelInfoList">
         <ActionPanel title.translate="Information" minWidth="200" initialWidth="250" icon="'fa fa-info'">
+            <button t-if="visitorPersona?.type === 'partner'" class="btn btn-primary mt-1" t-on-click="openVisitorContact" title="View Contact">
+                View Contact
+            </button>
```

Comment:
As this is a link to a new page, it would be better as a real link with `a` and `href`, it can keep the custom on-click, but if the href is correct it also allows ctrl+click or middle+click to open the link in a new tab, or right click to copy the link.